### DISCUSSION TITLE
Python version compatibility

### DISCUSSION THREAD

**Discussion by Jushan-Randy-Chen** [USC-ACTLab/crazyswarm#720] [Q&A]
Do I have to use Python 3.7 to install along side ROS Noetic? Can I use a newer version of Python?

**Comment by knmcguire** [comment]
You can try and see? But the fact that the installation instructions say 3.7 is not necessarily that it doesn't work with versions higher with that, but that it is no longer actively tested for any other version. 

So just go ahead and try a new version, but just be aware that if there are any errors, you probably will need to do some googling to fix it in the code yourself.
